spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'UsersControllers', type: :request do
+  describe 'Users controller testing' do
+    it 'GET /' do
+      get '/'
+      expect(response).to have_http_status(:ok)
+      expect(response).to render_template('users/index')
+      expect(response.body).to include('<h1>View to show users</h1>')
+    end
+
+    it 'GET /users/:id' do
+      get '/users/239'
```

Comment:
- [ ] Please kindly make sure all your tests are passing, I get an error that says this `id` is not found 🙃 🙃 

<img width="944" alt="Screen Shot 2023-01-23 at 20 47 47" src="https://user-images.githubusercontent.com/74501523/214113697-39a7aa82-5493-4899-bd89-e5d923aa36f2.png">